apps/test/unit/lib/ui/SettingsCogTest.js

```diff
@@ -62,4 +64,72 @@ describe('SettingsCog', () => {
       done();
     }, 0);
   });
+
+  describe('menu items', () => {
+    describe('manage assets', () => {
+      it('opens the asset manager and calls handleManageAssets when clicked', () => {
+        const handleManageAssets = sinon.spy();
+        const wrapper = mount(
+          <ManageAssets onClick={handleManageAssets}/>
+        );
+        expect(wrapper.text()).to.equal(msg.manageAssets());
+
+        expect(handleManageAssets).not.to.have.been.called;
+        wrapper.simulate('click');
+        expect(handleManageAssets).to.have.been.calledOnce;
+      });
+    });
+
+    describe('maker toggle', () => {
+      beforeEach(() => {
+        sinon.stub(maker, 'isAvailable');
+        sinon.stub(maker, 'isEnabled');
+      });
+
+      afterEach(() => {
+        maker.isEnabled.restore();
+        maker.isAvailable.restore();
+      });
+
+      it('renders with enable maker option if maker is available and disabled', () => {
+        maker.isAvailable.returns(true);
+        maker.isEnabled.returns(false);
+        const wrapper = mount(
+          <ToggleMaker/>
+        );
+        expect(wrapper.text()).to.include(msg.enableMaker());
+      });
+
+      it('renders with enable maker option if maker is available and disabled', () => {
+        maker.isAvailable.returns(true);
+        maker.isEnabled.returns(true);
+        const wrapper = mount(
+          <ToggleMaker/>
+        );
+        expect(wrapper.text()).to.include(msg.disableMaker());
+      });
+
+      it('hides maker toggle if maker is not available', () => {
+        maker.isAvailable.returns(false);
+        const wrapper = mount(
+          <ToggleMaker/>
+        );
+        expect(wrapper).to.be.blank;
+      });
+
+      it('calls project.toggleMakerEnabled and handleToggleMaker when clicked', () => {
```

Comment:
Whoops!  It used to test this, before I removed the global dependency in the component!